nb_train.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+import string
+import re
+import nltk
+import random
+import pickle
+from sklearn.model_selection import train_test_split
+from nltk.corpus import stopwords
+from zipfile import ZipFile
+
+with ZipFile('imdb ratings.zip') as zf:
```

Comment:
Hi Nick, I don't quite get this. I'm thinking the path to the zip file would look something like "./folder_name/zipfile", but wouldn't I still need to update that path if I changed the location of the zip file on my computer? 

I looked at your code from your project and saw you had constants at the top like YELP_PREFIX = "../pickle/yelp/balanced/" and PATH_TO_CLASSIFIER = YELP_PREFIX + "nb_multinomial_classifier.p". Are you suggesting I do something similar here? 

Another confusion I have is, how should I make my program executable and these file paths accessible for people if I am only uploading my code and not the data files to github?

Reply:
When you pass a location to a file as a [command line argument](https://realpython.com/python-command-line-arguments/), you pass an [absolute path](http://www.differencebetween.net/technology/difference-between-absolute-and-relative-path/#:~:text=In%20simple%20words%2C%20an%20absolute,directory%20you%20are%20working%20on.)
instead of a relative path.

That way, no matter where your program is located, you can pass the same file path and it will work.

Regarding your second point: that is actually the benefit of using command line arguments! If someone downloads your code onto their computer, they can put their zip file wherever they want and just pass the path to their file. If you hardcode the path to the file in your source code, it would only work on their computer if they were to put it in the right location.